src/pdfviewer/PDFDocument.cpp

```diff
@@ -2199,7 +2220,15 @@ int PDFWidget::pageStep()
  */
 int PDFWidget::gridCols(bool fromConfig) const
```

Comment:
like above
the concept from/to config and embedded/windowed needs to cleaned up